sites/drmax.js

```diff
@@ -0,0 +1,78 @@
+const { Scraper, postApiPeViitor } = require("peviitor_jsscraper");
+
+const generateJob = (job_title, job_link, city) => ({
+  job_title,
+  job_link,
+  country: "Romania",
+  city,
+});
+
+const getJobBatch = async (url) => {
+  const jobs = [];
+  const scraper = new Scraper(url);
+  const type = "HTML";
+  const soup = await scraper.get_soup(type);
+  const soupElements = soup.findAll("h2", { class: "awsm-job-post-title" });
+  const soupElements2 = soup.findAll("div", {
+    class: "awsm-job-specification-job-location",
+  });
+
+  for (let i = 0; i < soupElements.length; i += 1) {
+    soupElements[i].append(soupElements2[i]);
+  }
+  soupElements.forEach((el) => {
+    const job_title = el.find("a").text;
+    const job_link = el.find("a").attrs.href;
+    const city = el.find("span").text.trim();
+    const job = generateJob(job_title, job_link, city);
+    jobs.push(job);
+  });
+  return jobs;
+};
+
+const getJobs = async () => {
+  const allJobs = [];
+  const firstPageUrl =
+    "https://drmaxromania.com/oportunitati-de-cariera/?paged=1";
+  const scraper = new Scraper(firstPageUrl);
+  const type = "HTML";
+  const soup = await scraper.get_soup(type);
+  const soupElements3 = soup.findAll("a", { class: "page-numbers" });
+  const pagination = [];
+  soupElements3.forEach((el) => {
+    const page = el.text;
+    pagination.push(page);
+  });
+  const totalPages = Number(pagination[2]);
+  for (let page = 1; page <= totalPages; page += 1) {
+    const pageUrl = `https://drmaxromania.com/oportunitati-de-cariera/?paged=${page}`;
+    const pageJobs = await getJobBatch(pageUrl);
+    allJobs.push(...pageJobs);
+  }
+  return allJobs;
+};
+
+const getParams = () => {
+  const company = "Dr.Max";
```

Comment:
company name must be the same as the file name
```suggestion
<img width="1440" alt="Screenshot 2023-07-28 at 16 23 02" src="https://github.com/peviitor-ro/scrapers.js/assets/67306273/39d2e28e-e980-43d1-bb28-b8a5e74b5726">

  const company = "DrMax";
```